pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -57,9 +60,12 @@ func (t *ofTable) UpdateStatus(flowCountDelta int) {
 
 	if flowCountDelta < 0 {
 		t.flowCount -= uint(-flowCountDelta)
+
```

Comment:
I think this blank line doesn't help readability?

Reply:
Added by mistake. Done.